js/app/pages/Form.html

```diff
@@ -275,6 +275,13 @@
     }
     app.modal.handleStudentNote(student);
   },
+  isLocked: function() {
+    return app.pages.form.locked;
+  },
+  lock: function() {
+    app.pages.form.locked = true;
+  },
+  locked: false,
```

Comment:
What is this doing here? If it's an element, shouldn't it be up in the elements section?